Packs/CommonScripts/Scripts/script-DeleteContext.yml

```diff
@@ -54,6 +48,39 @@ script: >
               keysToKeepObj[keysToKeep[i]] = value;
           }
       }
+      return keysToKeepObj;
+  }
+
+  function createContext(keysToKeepObj) {
```

Comment:
avoid name conflicts.
there is already a function with that name in CommonServer.